diff --git a/dist/components/runtime/lazy-hydrated-component.js b/dist/components/runtime/lazy-hydrated-component.js
index e7da5c51f62e8a765c1a3bf905b971f53a23827b..8d67a0be5a9f399a62c7a8a1e4ab9cc478676d18 100644
--- a/dist/components/runtime/lazy-hydrated-component.js
+++ b/dist/components/runtime/lazy-hydrated-component.js
@@ -20,7 +20,7 @@ function defineLazyComponent(props, defineStrategy) {
       const onVnodeMounted = () => {
         ctx.emit("hydrated");
       };
-      return () => h(comp, mergeProps(ctx.attrs, { onVnodeMounted }));
+      return () => h(comp, mergeProps(ctx.attrs, { onVnodeMounted }), ctx.slots);
     }
   });
 }
